src/client/server.ts

```diff
@@ -0,0 +1,70 @@
+import { type CloudEvent, type CloudEventV1, HTTP, type Message } from 'cloudevents';
+import type { FastifyInstance, FastifyPluginOptions } from 'fastify';
+import type { BaseLogger } from 'pino';
+import env from 'env-var';
+
+import { makeFastify } from '../utilities/fastify/server.js';
+import { HTTP_STATUS_CODES } from '../utilities/http.js';
+import type { PluginDone } from '../utilities/fastify/PluginDone.js';
+import type { FastifyTypedInstance } from '../utilities/fastify/FastifyTypedInstance.js';
+
+import type { Sink } from './Sink.js';
+import { QueueProblemType } from './QueueProblemType.js';
+import exampleSink from './sinks/example.sink';
+
+const SINK_BY_TYPE: { [type: string]: Sink } = {
+  'example': exampleSink,
+};
```

Comment:
As discussed yesterday on the call, we should keep this simple and not borrow the sink mapping mini-framework we used in the Authority server.

The only event type this server will ever process are outgoing messages.